src/main/java/hlf/java/rest/client/service/impl/NetworkStatusImpl.java

```diff
@@ -399,6 +399,10 @@ public ResponseEntity<ClientResponseModel> addOrgToChannel(
         UpdateChannelConfiguration updateChannelConfiguration = new UpdateChannelConfiguration();
         updateChannelConfiguration.setUpdateChannelConfiguration(
             configUpdate.toByteString().toByteArray());
+        System.out.println(user.getName());
+        System.out.println(user.getRoles());
+        System.out.println(user.getEnrollment());
```

Comment:
Remove these. If there is a valid reason to display these in the runtime trace, log it with DEBUG level